detekt-rules-style/src/test/kotlin/io/gitlab/arturbosch/detekt/rules/style/ForbiddenNamedParamSpec.kt

```diff
@@ -53,7 +53,12 @@ class ForbiddenNamedParamSpec(val env: KotlinEnvironmentContainer) {
             TestConfig(METHODS to listOf("kotlin.io.println"))
         ).lintWithContext(env, code)
         assertThat(findings).hasSize(3)
-        assertThat(findings).hasTextLocations(17 to 38, 43 to 63, 68 to 90)
+        assertThat(findings).element(0)
+            .hasTextLocation(17 to 38)
+        assertThat(findings).element(1)
+            .hasTextLocation(43 to 63)
+        assertThat(findings).element(2)
+            .hasTextLocation(68 to 90)
```

Comment:
Is this maybe what you were looking for? Probably we could add some more syntactic kotlin sugar.

```suggestion
        assertThat(findings).satisfiesExactlyInAnyOrder(
            { finding -> assertThat(finding).hasTextLocation(17 to 38) },
            { finding -> assertThat(finding).hasTextLocation(43 to 63) },
            { finding -> assertThat(finding).hasTextLocation(68 to 90) }
        )
```

Reply:
😱 Does that exist out of the box??? That's exactly what we need, I think.

Reply:
Yes, the assertj API surface is so large that it is hard to find the right method.

Reply:
I guess that something like this would be even better.
```kotlin
        assertThat(findings).satisfiesExactlyInAnyOrder(
            { it.hasTextLocation(17 to 38) },
            { it.hasTextLocation(43 to 63) },
            { it.hasTextLocation(68 to 90) },
        )
```

Reply:
@marschwar I'm going to move from `element` to this API. It's way, way better. But what do you think about merging this first and then I make a PR removing all the usages of `element`. I think that the diff would be more clear. And it would also help me with the regex.

TL;DR: I'm asking to merge this first and then I'm going to create a PR with the port from `element` to `satisfiesExactlyInAnyOrder`.

Reply:
Yes, let's do that.